MovingBlocks/CrashReporter
# build: bump gradle 6.9.2 -> 8.2.1

Relates to https://github.com/MovingBlocks/Terasology/pull/5109
Relates to https://github.com/MovingBlocks/Terasology/pull/4653

### Contains

* Updating gradle from 6.9.2 -> 8.2.1
* Updating gradle's files which became invalid

### How to test

1. Go into your Terasology workspace
2. Run `groovyw lib get CrashReporter`
3. In `libs/CrashReporter` check out this branch
4. Start Terasology and provoke a crash (e.g. by checking out commit 97f180b1513151ee787d2b0d78cad5c1fc861a2c in the engine repo)
5. Check that CrashReporter opens and works fine

### Remarks

Java 20 is not yet supported, see https://github.com/gradle/gradle/issues/23488.
